blacklight-cornell/spec/controllers/search_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+# spec/controllers/search_controller_spec.rb
+require "rails_helper"
+
+RSpec.describe SearchController, type: :controller do
+  describe "SearchController.transform_query" do
+    it "transforms the query correctly" do
+      query = "going fishing"
+      transformed_query = SearchController.transform_query(query)
+      expect(transformed_query).to eq('("going" AND "fishing") OR phrase:"going fishing"')
+    end
+
+    it "handles already-quoted queries correctly" do
+      # pending("Pending implementation of handling already-quoted queries")
+      query = '"going fishing"'
+      transformed_query = SearchController.transform_query(query)
+      expect(transformed_query).to eq("\"going fishing\"")
```

Comment:
Not sure if this transform_query works just for the d&a solr index, but in case it does:

For simple and advanced search, this would become: `quoted:"going fishing"`

Reply:
That is what I changed it _FROM_. I'll wait until your PR is merged and revisit.